homeassistant/components/vulcan/config_flow.py

```diff
@@ -0,0 +1,342 @@
+                    errors["base"] = "invalid_token"
+                elif str(err) == "Expired token.":
+                    errors["base"] = "expired_token"
+                elif str(err) == "Invalid PIN.":
+                    errors["base"] = "invalid_pin"
+                else:
+                    errors["base"] = "unknown"
+                    _LOGGER.error(err)
+            except RuntimeError as err:
+                if str(err) == "Internal Server Error (ArgumentException)":
+                    errors["base"] = "invalid_symbol"
+                else:
+                    errors["base"] = "unknown"
+                    _LOGGER.error(err)
+            except ClientConnectionError as err:
+                errors["base"] = "cannot_connect"
+                _LOGGER.error("Connection error: %s", err)
+            except Exception:  # pylint: disable=broad-except
+                _LOGGER.exception("Unexpected exception")
+                errors["base"] = "unknown"
+            if not errors:
+                account = credentials["account"]
+                keystore = credentials["keystore"]
+                client = Vulcan(keystore, account)
+                students = await client.get_students()
+                await client.close()
+                existing_entries = []
+                for entry in self.hass.config_entries.async_entries(DOMAIN):
+                    existing_entries.append(entry)
+                for student in students:
+                    for entry in existing_entries:
+                        if str(student.pupil.id) == str(entry.data["student_id"]):
+                            self.hass.config_entries.async_update_entry(
+                                entry,
+                                title=f"{student.pupil.first_name} {student.pupil.last_name}",
+                                data={
+                                    "student_id": str(student.pupil.id),
+                                    "keystore": keystore.as_dict,
+                                    "account": account.as_dict,
+                                },
+                            )
+                            await self.hass.config_entries.async_reload(entry.entry_id)
+                return self.async_abort(reason="reauth_successful")
+
+        return self.async_show_form(
+            step_id="reauth",
+            data_schema=vol.Schema(LOGIN_SCHEMA),
+            errors=errors,
+        )
+
+
+class VulcanOptionsFlowHandler(config_entries.OptionsFlow):
+    """Config flow options for Uonet+ Vulcan."""
+
+    def __init__(self, config_entry):
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(self, user_input=None):
+        """Manage the options."""
+        errors = {}
+
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        options = {
+            vol.Optional(
+                CONF_SCAN_INTERVAL,
```

Comment:
We don't allow an option for scan interval. Every integration supporting config entries supports disabling automatic polling and then the user can automate polling manually with the entity service `homeassistant.update_entity.

Please remove this option.